app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PostsController < ActionController
+  def index; end
+  def show; end
+end
```

Comment:
- [ ] Let's ensure that `PostController` inherits from `ApplicationController`, not `ActionController`, and our `PostController` class should be similar to:

```ruby
# app/controllers/posts_controller.rb
class PostsController < ApplicationController
  
  def index; end

  def show; end

end
```

Reply:
well resolved! ✔️